### PR TITLE
[locale] bn: Better month and weekday names

### PR DESCRIPTION
**Pull request by shakib609**
@kaushikgandhi See if everything's okay?

### CODE REVIEW

**Comment by icambron** [comment]
I have no idea if the changes are right, but you would need to fix the tests too.

**Comment by shakib609** [comment]
@icambron I just changed some translations. No other code modifications. I really don't get it why the test fails?

**Comment by kaushikgandhi** [comment]
Commit looks good specially that February spelling mistake that u corrected
@shakib609 ... But I didn't like the changes in weekdaysMin I think the
current one is better. The point of having that attribute is to have short
abbreviations.

**Comment by kaushikgandhi** [comment]
Please fix the test cases if the code changes the output expectation also changes. So tests will change

**Comment by icambron** [comment]
@shakib609 the translations have tests. You wan [this one](https://github.com/moment/moment/blob/develop/src/test/locale/bn.js).

**Comment by shakib609** [comment]
Thanks @icambron I'm fixing the tests.
@kaushikgandhi If you don't like it I can revert back the `weekdaysMin`.

**Comment by kaushikgandhi** [comment]
Sure please revert that back.

**Comment by shakib609** [comment]
Ok @kaushikgandhi

**Comment by shakib609** [comment]
@kaushikgandhi Actually I think the `weekdaysMin` should be at least 2 letters? Don't you think it would be more readable that way?

**Comment by shakib609** [comment]
@kaushikgandhi @icambron Is there anything wrong with my commits? Please let me know. I'll fix it ASAP.

**Comment by kaushikgandhi** [comment]
@icambron everything looks ok. You can merge the commit.

File: locale/bn.js [hunk 1]
```diff
@@ -36,11 +36,11 @@
     };
 
     var bn = moment.defineLocale('bn', {
-        months : 'জানুয়ারী_ফেবুয়ারী_মার্চ_এপ্রিল_মে_জুন_জুলাই_অগাস্ট_সেপ্টেম্বর_অক্টোবর_নভেম্বর_ডিসেম্বর'.split('_'),
```
Comment (ichernev) [inline locale/bn.js]: do not change this file, it is autogenerated

**Comment by ichernev** [comment]
This will be merged in next release. Thank you @shakib609, @kaushikgandhi and @icambron !

**Comment by shakib609** [comment]
@ichernev ok. I'll revert `locale/bn.js`.

**Comment by mj1856** [comment]
@shakib609 - please rebase to one commit, reflecting only changes to the `src/locale/bn.js` and `src/test/locale/bn.js` files.  Thanks.

**Comment by shakib609** [comment]
Any idea why this is happening.
I'm only changing the translations as you can see.
```
Module: relative time Test: custom rounding

Round down towards the nearest year

    at Object.<anonymous> (/home/travis/build/moment/moment/build/umd/test/moment/relative_time.js:455:16)

Actual value:
in 1.6250000115740741 days

Expected value:
in 1.625 days
```

**Comment by mj1856** [comment]
Intermittent floating point error in the test, we need to fix, but not related to your change.  I re-ran the CI build and passes now.

**Comment by ichernev** [comment]
Merged in 8cba0f59679bb08fd9e46d71a1d08b8ad70871fe

**Pull request closed without merging**